yarn.lock

```diff
@@ -1147,7 +1147,7 @@
   dependencies:
     "@hapi/hoek" "9.x.x"
```

Comment:
`core-p2p` and `core-webhooks` were not updated to `@hapi/boom@9.1.4` and pulled in older `@hapi/boom@9.0.0` due to strict version match (using `9.0.0` not `^9.0.0).